crates/uv/tests/it/python_install.rs

```
@@ -2255,6 +2255,56 @@ fn python_install_broken_link() {
     });
 }
 
+#[test]
+fn python_install_default_prerelease() {
+    // Test that --default works with pre-release versions (e.g., 3.15.0a1).
+    // This test verifies the fix for issue #16696 where --default didn't create
+    // python.exe and python3.exe links for pre-release versions.
+    let context: TestContext = TestContext::new_with_versions(&[])
+        .with_filtered_python_keys()
+        .with_filtered_exe_suffix()
+        .with_managed_python_dirs()
+        .with_python_download_cache();
+
+    // Try to install Python 3.15, which currently only exists as a pre-release (3.15.0a1).
+    // If 3.15 is not available, this test will be skipped naturally.
+    let Ok(output) = context
+        .python_install()
+        .arg("--default")
+        .arg("--preview-features")
+        .arg("python-install-default")
+        .arg("3.15")
+        .output()
+    else {
+        return; // Skip test if command fails
+    };
```

We should `.assert().success()` here instead. 3.15 _is_ available, we shouldn't let the test silently fail.

---

Yup, fixed this!